spark/src/main/scala/org/apache/spark/sql/delta/TableFeature.scala

```diff
@@ -682,7 +682,7 @@ object InCommitTimestampTableFeature
  * feature exists
  */
 object VacuumProtocolCheckTableFeature
-  extends ReaderWriterFeature(name = "vacuumProtocolCheck-dev")
+  extends ReaderWriterFeature(name = "vacuumProtocolCheck")
```

Comment:
isnt there a change needed in the tests to make sure that you dont have to enable some testing-only internal flag to use this feature?

Reply:
The change to ensure the feature is available without enabling a testing-only internal flag is included in the current file by moving it from testing-set to prod-set. 

```
if (DeltaUtils.isTesting) {
      features ++= Set(
        TestLegacyWriterFeature, ....
```

Reply:
I see. thats how it works. good.